projects/commudle-admin/src/app/feature-modules/user-chats/components/chats-container/chats-container.component.ts

```diff
@@ -53,30 +55,46 @@ export class ChatsContainerComponent implements OnInit, OnDestroy {
     }
 
     // If clicked on messaging button in any user profile
-    this.subscriptions.push(this.userChatsService.followerIdChange$.subscribe(data => this.createChat(data)));
+    this.subscriptions.push(this.userChatsService.followerIdChange$.subscribe((data) => this.createChat(data)));
+
+    //new chat
+    this.checkNewMessage();
   }
 
   ngOnDestroy() {
-    this.subscriptions.forEach(value => value.unsubscribe());
+    this.subscriptions.forEach((value) => value.unsubscribe());
```

Comment:
@The-Vector-007 let's please use more meaningful names here instead of value. Probably we could call it 'subscription' or 'sub' or....

Reply:
done!